test/e2e/framework/framework.go

```diff
@@ -63,12 +63,16 @@ func (f *Framework) BeforeEach() {
 	By("Creating a kubernetes client")
 	kubeConfig, err := LoadConfig(TestContext.KubeConfig, TestContext.KubeContext)
 	Expect(err).NotTo(HaveOccurred())
+	kubeConfig.QPS = 50
```

Comment:
What were the default values?

Reply:
```
	DefaultQPS   float32 = 5.0
	DefaultBurst int     = 10
```

Reply:
5 and 10, and I was copying the values from upstream, as I feel upstream would pick safe values.

Maybe it has since changed.

Reply:
On second look, they use 20 and 50, so I'm guessing I read those values in commentary somewhere, not the PR.